internal/aws/ssm_client.go

```diff
@@ -172,15 +172,15 @@ func (s *SSMClient) CreateActivation(ctx context.Context, description string, in
 		})
 	}
 
-	// Set expiration to 24 hours from now
-	expirationTime := time.Now().Add(24 * time.Hour)
+	// Set expiration to 5 minutes from now
+	expirationTime := time.Now().Add(5 * time.Minute)
```

Comment:
non-blocking: you can set a default value, but pipe through  an override from `accessStrategy` please